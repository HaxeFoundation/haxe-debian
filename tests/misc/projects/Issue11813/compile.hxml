-m Main
-w +WUnsafeEnumEquality
--interp