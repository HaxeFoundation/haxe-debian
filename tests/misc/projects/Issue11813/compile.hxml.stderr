Main.hx:19: characters 6-22 : Warning : (WUnsafeEnumEquality) Equality operations on this enum might lead to unexpected results because some constructors have arguments
Main.hx:20: characters 6-22 : Warning : (WUnsafeEnumEquality) Equality operations on this enum might lead to unexpected results because some constructors have arguments